recipes/bioconductor-dexseq/pythonpath.patch

```diff
@@ -0,0 +1,16 @@
+--- inst/python_scripts/dexseq_count.py	2016-10-17 15:19:41.000000000 -0500
++++ inst/python_scripts/dexseq_count.py.new	2017-06-16 14:14:38.000000000 -0500
+@@ -1,3 +1,5 @@
++#!/opt/anaconda1anaconda2anaconda3/bin/python
++
+ import sys, itertools, optparse, warnings
+ 
+ optParser = optparse.OptionParser( 
+--- inst/python_scripts/dexseq_prepare_annotation.py	2016-10-17 15:19:41.000000000 -0500
++++ inst/python_scripts/dexseq_prepare_annotation.py.new	2017-06-16 14:14:55.000000000 -0500
+@@ -1,3 +1,5 @@
++#!/opt/anaconda1anaconda2anaconda3/bin/python
```

Comment:
Well my reasoning was that since all scripts (`python`, `R`, `perl`, etc.) that are installed as part of normal package installations are treated that way by `conda-build`, it's good to match that behavior.  IMHO, using `/usr/bin/env foo` is contrary to the idea of creating reproducible, portable installs as it potentially allows local variations in a user's environment to alter or break functionality.  That being said, I'm happy to follow whatever the Core team would like to establish as best practice.